apps/spark/build.gradle

```diff
@@ -142,3 +147,12 @@ shadowJar {
 // By default shadow doesn't configure the build task to depend on the shadowJar task.
 tasks.build.dependsOn tasks.shadowJar
 
+test {
+    if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
```

Comment:
For my understanding, what are the usage of these args?

Reply:
at build time, if the java version is 9 or later we add these flag, these flags were introduced in java 9 when reflection API to change visibility of methods was removed